archetypes/adrs.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I just checked and this file has no effect. Running `hugo new content adr/something.md` doesn't utilize this template. It just looks like it does because it matches the default template.

We have to rename this to `adr.md` for it to take effect. Also, we should add more fields like `decision`, `status`, and `categories` to this template.

Reply:
Further, we should also include template content here. It is supported as per https://gohugo.io/content-management/archetypes/#include-content

Reply:
Yes, you are right. I will make these changes accordingly.